### PR TITLE
Tweaks Poison Kit

### PR DESCRIPTION
**Pull request by ArcLumin**
Removes neurotoxin and formaldehyde from the poison kit, replaces with wasting toxin and sleep beer.

### CODE REVIEW

**Comment by ArcLumin** [comment]
Lemme know if I need to boost the TC cost for this tweak

**Comment by KayeArray** [comment]
Wasting toxin was nerfed from that OP bullshit it used to be right?

**Comment by ArcLumin** [comment]
Yeah. It's still somewhat strong, but FAR from the strongest thing in the poison kit.

**Comment by KayeArray** [comment]
I was just wondering if it still overdoses you on tons of venom and kills you in like 5 seconds flat but I think that got changed

**Comment by ArcLumin** [comment]
It gives less toxin damage than most stuff. It's about bleach level toxin damage

What it mostly does is remove blood from your system.

**Comment by KayeArray** [comment]
Yeah that's pretty fine, I support this

**Comment by GuyonBroadway** [comment]
What I found funny about wasting toxin is it broke down into 10u of venom which then breaks down into 10u of histamine meaning you very quickly end up with three deadly chems in your system at once.

**Comment by ArcLumin** [comment]
Thats old wasting toxin

**Comment by GuyonBroadway** [comment]
And thank god that's gone, 5u of the fukken stuff could easily kill you. A Wasting toxin smoke grenade could depopulate large areas of the station.

**Comment by Pyko1** [comment]
Made an actually issue about oddy chem dupe.
https://github.com/HippieStationCode/HippieStation13/issues/2991

**Comment by ArcLumin** [comment]
We hadn't even talked about oddy yet, hold your horses lad

**Comment by Pyko1** [comment]
well already talked about on other chem releated PR's might be better to have oddy issues in single issue and handle them there.

File: code/modules/food&drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -168,6 +168,9 @@
 	desc = "Beer. In space."
 	icon_state = "beer"
 	list_reagents = list("beer" = 30)
+	
+/obj/item/weapon/reagent_containers/food/drinks/beer/sleep
```
Comment (Kierany9) [inline code/modules/food&drinks/drinks/drinks.dm]: You sure that the obj named sleep won't do anything fucky?

**Comment by doobiedoo23** [comment]
+1 once it's been reviewed

File: code/modules/food&drinks/drinks/drinks.dm (hunk 1 above)
Comment (ArcLumin) [reply code/modules/food&drinks/drinks/drinks.dm]: Yes, I'm sure. I think.

Comment (ArcLumin) [reply code/modules/food&drinks/drinks/drinks.dm]: I'll change it just in case.

**Comment by ArcLumin** [comment]
This has been reviewed.

**Comment by SplergeBucket** [comment]
i'll go with a h+1??

**Comment by ghost** [comment]
Yeah sure, although these aren't even the worst things in the traitor chem shit.

**Pull request merged**